### PR TITLE
Word hatin!

### PR DESCRIPTION
**Pull request by threeSpeed**

### CODE REVIEW

**Comment by davetapley** [comment]
Hey @threeSpeed, is this ready for master?

**Comment by threeSpeed** [comment]
you da boss.

File: food.txt [hunk 1]
```diff
@@ -1 +1,6 @@
+sdfsd
+doing some work. getting paid.
+
 food is good too.
+
+Microsoft sucks!
```
Comment (davetapley) [inline food.txt:6]: @threeSpeed don't you agree?

Comment (threeSpeed) [inline food.txt:6]: Its after hours and Im drinking beer, so stop talking about work.

**Pull request merged**